web/src/utils/backend-client/admin/admin.ts

```diff
@@ -615,6 +615,51 @@ export const useListProviderArgOptions = <TError = ErrorType<ErrorResponseRespon
   }
 }
 
+/**
+ * Returns the filters for a particular Access Provider argument. 
+ * @summary List provider arg filters
+ */
+export const listProviderArgFilters = (
+    providerId: string,
+    argId: string,
+    filterId: string,
+ options?: SecondParameter<typeof customInstance>) => {
+      return customInstance<ArgOptionsResponseResponse | void>(
+      {url: `/api/v1/admin/providers/${providerId}/args/${argId}/filters/${filterId}`, method: 'get'
+    },
+      options);
+    }
+  
+
+export const getListProviderArgFiltersKey = (providerId: string,
```

Comment:
starting to feel like this API call isn't required. And we can add the filter options value to the `listArgOptions` API itself. Thoughts?